MAUI/Calendar/footer.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Footer in .NET MAUI Calendar control | Syncfusion | Calendar
+description: Learn here all about Footer view feature of Syncfusion .NET MAUI Calendar (SfCalendar) control and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Footer in .NET MAUI Calendar (SfCalendar)
+You can customize all the properties of the Footer view using [FooterView]. By using this property, you can customize the Background, Height, DividerColor, TextStyle, ShowActionButtons and ShowTodayButton of the Calendar.
+
+## Customize the header height
+You can customize the footer height `Calendar` by using the `Height` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="calendar"
+                      View="Month">
+            <calendar:SfCalendar.FooterView>
+                <calendar:CalendarFooterView Height="70" ShowActionButtons="True" ShowTodayButton="True" />
+            </calendar:SfCalendar.FooterView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.calendar.FooterView.Height = 70;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Action buttons
+
```

Comment:
Already action and today button explained in. getting started - So please remove that content

<img width="361" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/46158936/326a6d69-60b6-4960-914c-9b793e8770c9">